src/fontra/views/fontinfo/panel-development-status-definitions.js

```diff
@@ -0,0 +1,323 @@
+          this.postChange.bind(this),
+          this.setupUI.bind(this)
+        )
+      );
+    }
+
+    this.panelElement.innerHTML = "";
+    this.panelElement.style = `
+    gap: 1em;
+    `;
+    this.panelElement.appendChild(
+      html.input({
+        type: "button",
+        style: `justify-self: start;`,
+        value: "New status definition",
+        onclick: (event) => this.newStatusDef(),
+      })
+    );
+    this.panelElement.appendChild(statusDefsContainer);
+    this.panelElement.focus();
+  }
+
+  async newStatusDef(statusDef = undefined) {
+    const statusFieldDefinitions =
+      this.fontController.customData["fontra.sourceStatusFieldDefinitions"];
+    const nextStatusValue = !statusFieldDefinitions
+      ? 0
+      : statusFieldDefinitions
+          .map((statusDef) => statusDef.value)
+          .sort()
+          .pop() + 1;
+
+    // fix duplicates
+    if (
+      statusDef &&
+      statusFieldDefinitions.some((statusDef) => statusDef.value === statusDef.value)
+    ) {
+      // Status definition with value already exists,
+      // changed value to next available value.
+      statusDef = {
+        ...statusDef,
+        value: nextStatusValue,
+      };
+    }
+
+    // fix isDefault
+    if (statusDef && statusFieldDefinitions.some((statusDef) => statusDef.isDefault)) {
+      // Status definition with isDefault already exists,
+      // changed value to false.
+      delete statusDef["isDefault"];
+    }
+
+    if (!statusDef) {
+      const defaultStatuses =
+        defaultStatusFieldDefinitions["fontra.sourceStatusFieldDefinitions"];
+      statusDef = defaultStatuses.find(
+        (statusDef) => statusDef.value == nextStatusValue
+      );
+    }
+
+    if (!statusDef) {
+      // No status definition provided or found.
+      // Use default.
+      statusDef = {
+        color: [1, 0, 0, 1],
+        label: `Status definition ${nextStatusValue}`,
+        value: nextStatusValue,
+      };
+    }
+
+    const undoLabel = `add status definition ${statusDef.value} '${statusDef.label}'`;
+    const root = { customData: this.fontController.customData };
+    const changes = recordChanges(root, (root) => {
+      if (!statusFieldDefinitions) {
+        root.customData["fontra.sourceStatusFieldDefinitions"] = [];
+      }
+      root.customData["fontra.sourceStatusFieldDefinitions"].push(statusDef);
+    });
+    if (changes.hasChange) {
+      this.postChange(changes.change, changes.rollbackChange, undoLabel);
+      this.setupUI();
+    }
+  }
+}
+
+function getStatusFieldDefinitions(fontController) {
+  const statusFieldDefinitions =
+    fontController.customData["fontra.sourceStatusFieldDefinitions"];
+  if (statusFieldDefinitions) {
+    return statusFieldDefinitions;
+  }
+  return [];
+}
+
+addStyleSheet(`
+.fontra-ui-font-info-status-definitions-panel-status-def-box {
+  background-color: var(--ui-element-background-color);
+  border-radius: 0.5em;
+  padding: 1em;
+  cursor: pointer;
```

Comment:
The whole box now causes the cursor to be "pointer", but since you don't support drag and drop, this makes no sense: the user will now think it can drag the box while it can't.

Reply:
I missed to remove it. Thanks for finding it.